docs/user/airbrakes.rst

```diff
@@ -0,0 +1,431 @@
+    
+Our example ``controller_function`` will deploy the air brakes when the rocket
+reaches 1500 meters above the ground. The deployment level will be function of the
+vertical velocity at the current time step and of the vertical velocity at the
+previous time step.
+
+Also, the controller function will check for the burnout of the rocket's motor 
+and only deploy the air brakes if the rocket has reached burnout. 
+
+Then, a limitation for the opening/closing speed of the air brakes will be set.
+The air brakes deployment level will not be able to change faster than 20% per
+second, in our case.
+
+Lets define the controller function:
+
+.. jupyter-execute::
+
+    def controller_function(
+        time, sampling_rate, state, state_history, observed_variables, air_brakes
+    ):
+        # state = [x, y, z, vx, vy, vz, e0, e1, e2, e3, wx, wy, wz]
+        altitude_ASL = state[2]
+        altitude_AGL = altitude_ASL - env.elevation
+        vx, vy, vz = state[3], state[4], state[5]
+        mach_number = (vx**2 + vy**2 + vz**2) ** 0.5 / env.speed_of_sound(
+            altitude_AGL - env.elevation
+        )
+
+        # Get previous state from state_history
+        previous_state = state_history[-1]
+        previous_vz = previous_state[5]
+
+        # If we wanted to we could get the returned values from observed_variables:
+        # returned_time, deployment_level, drag_coefficient = observed_variables[-1]
+
+        # Check if the rocket has reached burnout
+        if time > Pro75M1670.burn_out_time:
+            # If below 1500 meters above ground level, air_brakes are not deployment
+            if altitude_AGL < 1500:
+                air_brakes.set_deployment_level(0)
+
+            # Else calculate the deployment level
+            else:
+                # Controller logic
+                new_deployment_level = (
+                    air_brakes.deployment_level + 0.1 * vz + 0.01 * previous_vz**2
+                )
+
+                # Limiting the speed of the air_brakes to 0.2 per second
+                # Since this function is called every 1/sampling_rate seconds
+                # the max change in deployment level per call is 0.2/sampling_rate
+                max_change = 0.2 / sampling_rate
+                if new_deployment_level > air_brakes.deployment_level + max_change:
+                    new_deployment_level = air_brakes.deployment_level + max_change
+                elif new_deployment_level < air_brakes.deployment_level - max_change:
+                    new_deployment_level = air_brakes.deployment_level - max_change
+
+                air_brakes.set_deployment_level(new_deployment_level)
+
+            # Return variables of interest to be saved in the observed_variables list
+            return (
+                time,
+                air_brakes.deployment_level,
+                air_brakes.drag_coefficient(air_brakes.deployment_level, mach_number),
+            )
```

Comment:
I've made some "simplifications" to these code blocks when I updated the air brakes notebook. Do you mind checking the notebook and evaluating if it is worth to implement that in this page as well?

I'm sorry for keeping this short.